lib/screens/explore_screens/manage_invites/interactor/mappers/get_invites_state_mapper.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:seeds/datasource/remote/model/invite_model.dart';
+import 'package:seeds/datasource/remote/model/profile_model.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/screens/explore_screens/manage_invites/interactor/usecases/load_invites_use_case.dart';
+import 'package:seeds/screens/explore_screens/manage_invites/interactor/viewdata/InvitesItemsData.dart';
+import 'package:seeds/screens/explore_screens/manage_invites/interactor/viewmodels/manage_invites_state.dart';
+
+class GetInvitesStateMapper extends StateMapper {
+  ManageInvitesState mapResultToState(ManageInvitesState currentState, InvitesDto? invitesDto) {
+    if (invitesDto == null) {
+      return currentState.copyWith(pageState: PageState.failure);
```

Comment:
Is fine this fail with no message?

Reply:
this is not finished. next pr takes care of UI and error